src/devtools/views/tree/node.js

```diff
@@ -45,23 +36,13 @@ function Node({ action, x, y, index, send }) {
         y={offsetY}
         fontSize="14"
         textAnchor="start"
-        fill={action.disabled ? 'gray' : 'white'}
+        fill={
+          action.id == head ? '#ffeca4' : action.disabled ? 'gray' : 'white'
```

Comment:
It's a bit more verbose, but i like to pull double ternaries out into if/else blocks. Could this be a separate function? Also what if you adjusted the opacity instead of the fill?